### PR TITLE
Bz#1816749

### PR DESCRIPTION
**Pull request by spetrosi**
Make CMG work for both upstream and downstream

### CODE REVIEW

File: guides/doc-Content_Management_Guide/topics/Managing_Custom_File_Type_Content.adoc [hunk 1]
```diff
@@ -99,14 +99,16 @@ Use this procedure to configure a repository in a directory on the base system w
...
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
 --enable={RepoRHEL7ServerSatelliteToolsProductVersion}
 ----
-+
+endif::[]
+
 . Install the Pulp Manifest package:
 +
 [options="nowrap" subs="+quotes,attributes"]
```
Comment (spetrosi) [inline guides/doc-Content_Management_Guide/topics/Managing_Custom_File_Type_Content.adoc:114]: Hi @ehelms, in Satellite, the python-pulp-manifest package is available in satellite-tools repository. Do you know where does this package come from in Foreman? I cannot find the exact repo that provides it.

The python-pulp-manifest package was added to satellite-tools as part of this bug:
 https://bugzilla.redhat.com/show_bug.cgi?id=1488515#c16

File: guides/doc-Content_Management_Guide/topics/Managing_Activation_Keys.adoc [hunk 2]
```diff
@@ -72,7 +72,7 @@ To create an activation key, complete the following steps:
 . If you want to set a limit, clear the *Unlimited hosts* check box, and in the *Limit* field, enter the maximum number of systems you can register with the activation key. If you want unlimited hosts to register with the activation key, ensure the *Unlimited Hosts* check box is selected.
 . In the *Description* field, enter a description for the activation key.
 . From the *Environment* list, select the environment to use.
-. From the *Content View* list, select a Content View to use. If you want to use this activation key to register hosts, the Content View must contain the {Project} Tools repository because it is required to install the `katello-agent`.
+. From the *Content View* list, select a Content View to use. If you want to use this activation key to register hosts, the Content View must contain the `{RepoRHEL7ServerSatelliteToolsProductVersion}` repository because it is required to install the `katello-agent`.
```
Comment (spetrosi) [inline guides/doc-Content_Management_Guide/topics/Managing_Activation_Keys.adoc:75]: Hi @mcorr, does this solution look elegant to you? The attribute for the satellite-tools repo exists, so I just reused this attribute in all three builds (foreman, foreman-deb, satellite).

**Comment by tahliar** [comment]
A few general comments; it's up to you whether you implement them now or save them for a later PR. I've tried to avoid repeating anything that's already in the review Gdoc. 

* The Introduction talks about Red Hat subscriptions/content, etc. Should we add some ifevals, or just take out "Red Hat" and let it be generic?

* Similarly, can we find a way to make references to the CDN more generic, or do we need to ifeval them? e.g. "Foreman imports content from the Red Hat Content Delivery Network (CDN)." Could we say "{Project} can import content from the Red Hat Content Delivery Network (CDN)", which is upstream-centric but not blatantly untrue for downstream either?

* This is a TODO for the future and not for this PR, but we may need to rethink the chapter structure to be less Red Hat-focused - Importing Red Hat Content, for example, includes sections that apply to repos in general, including custom ones/Foreman ones, which will be more common upstream.

* Any images that are clearly downstream branded - we should either hide them in the Foreman build or add the same note Melanie put in the Install guide https://raw.githubusercontent.com/theforeman/foreman-documentation/master/guides/common/modules/AWS_Deployment_Scenarios.adoc

* There are some RHEL 8 system purpose items in the Activation Keys chapter. From some of lzap's comments in the Managing Hosts review, it seems like we should hide these upstream?

**Comment by spetrosi** [comment]
>     * The Introduction talks about Red Hat subscriptions/content, etc. Should we add some ifevals, or just take out "Red Hat" and let it be generic?
Foreman has a lot to do with Red Hat subscriptions. People buy RHEL subs but not Satellite subs so they use Foreman to provision RHELs. So this is fine.
> 
>     * Similarly, can we find a way to make references to the CDN more generic, or do we need to ifeval them? e.g. "Foreman imports content from the Red Hat Content Delivery Network (CDN)." Could we say "{Project} can import content from the Red Hat Content Delivery Network (CDN)", which is upstream-centric but not blatantly untrue for downstream either?
Done.
> 
>     * This is a TODO for the future and not for this PR, but we may need to rethink the chapter structure to be less Red Hat-focused - Importing Red Hat Content, for example, includes sections that apply to repos in general, including custom ones/Foreman ones, which will be more common upstream. 
I see two separate chapters: Importing Red Hat Content and Importing Custom Content, both are related to upstream.
> 
>     * Any images that are clearly downstream branded - we should either hide them in the Foreman build or add the same note Melanie put in the Install guide https://raw.githubusercontent.com/theforeman/foreman-documentation/master/guides/common/modules/AWS_Deployment_Scenarios.adoc
Done
> 
>     * There are some RHEL 8 system purpose items in the Activation Keys chapter. From some of lzap's comments in the Managing Hosts review, it seems like we should hide these upstream?
I have tested it in my Foreman instance, and the Activation Key settings in Foreman include system purpose as well. I will need to ask lzap to elaborate on what he meant in his review of the Managing hosts guide.

**Comment by tahliar** [comment]
I'm happy with the current state of this PR. Sergei and I agreed to merge this one and open another PR later with additional changes.

File: guides/doc-Content_Management_Guide/topics/Managing_Custom_File_Type_Content.adoc (hunk 1 above)
Comment (spetrosi) [reply guides/doc-Content_Management_Guide/topics/Managing_Custom_File_Type_Content.adoc:114]: @ehelms, I have installed my own Foreman and found that this is provided by the pulp repo

**Pull request merged**